cg/apps/osticket.py

```diff
@@ -40,3 +50,10 @@ def open_ticket(self, name: str, email: str, subject: str, message: str) -> Opti
 
         LOG.error("res.text: %s, reason: %s", res.text, res.reason)
         raise TicketCreationError(res)
+
+    @staticmethod
+    def create_attachment(
+        order_dict: dict,
+        file_name: str = "order.json",
+    ) -> dict:
+        return {file_name: TEXT_FILE_ATTACH_PARAMS.format(content=json.dumps(order_dict))}
```

Comment:
Assuming OsTicket is quite a general API then it makes more sense to leave out business-logic colored names like order and such 
```suggestion
        content: dict,
        file_name: str,
    ) -> dict:
        return {file_name: TEXT_FILE_ATTACH_PARAMS.format(content=json.dumps(content))}
```